samples/nrf9160/http_application_update/prj.conf

```diff
@@ -0,0 +1,61 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Is it a copy-paste, or is Newlib really needed here?

Reply:
Fixed now.